wandb/sdk/wandb_run.py

```diff
@@ -79,6 +78,11 @@
     from .wandb_setup import _WandbSetup
     from wandb.apis.public import Api as PublicApi
 
+    from typing import TYPE_CHECKING
```

Comment:
This looks magical, this is just a constant inside of the `typing` module?

Reply:
Yep, magic, and the canonical way this is done with type checkers. 

I will rename wandb.TYPE_CHECKING eventually so it is not as easily confused for typing magic.